android/app/src/internalRelease/assets/airshipconfig.properties

```diff
@@ -0,0 +1,7 @@
+appKey = 55vypj0ARc6cN09MX7ogtQ
+appSecret = EsSaqbdLSvmyC6kSBFJCtQ
+inProduction = true
+
+# Notification Customization
+notificationIcon = ic_notification
+notificationAccentColor = #2EAAE2
```

Comment:
An improvement I wanted to make previously where this becomes the default file was rejected as we're going to soon replace this.